.gitignore

```diff
@@ -43,6 +45,10 @@ logs/
 # TypeScript cache
 *.tsbuildinfo
 
+# Docusaurus specific
+docs/.docusaurus/
```

Comment:
let's keep the docs gitignore stuff in the /docs/.gitignore file instead of moving it here? (like it previously was)